### ISSUE TITLE
Unable to use Grammarly desktop with Nota

### ISSUE THREAD

**Issue by ambitendencies** [notaapp/talk#326]
Like title states; Unable to use Grammarly desktop with Nota.

I'm on a M1 Max Macbook Pro.

**Comment by astoilkov** [comment]
Yeah. There is another user that reported this as well — https://github.com/notaapp/talk/discussions/317. I've started researching this.
